RELEASING.md

```diff
@@ -0,0 +1 @@
+> The release process involves signing built packages and binaries and thus must be kept private and not exposed publicly.
```

Comment:
I think the original wording could be misunderstood to mean that the packages and binaries must be kept private. How about this:

```suggestion
> Because the release process involves signing of output packages and binaries, the release process is kept private and not exposed publicly.
```

Reply:
@breedx-splk unfortunately this is copied 1:1 from the spec, so we should change the spec first :)

Reply:
awesome. heh.